src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:39211/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:39211/project/issues?id=sample&pullRequest=6110&issues=AWen1cRZC7Ctzk_hAYQI&open=AWen1cRZC7Ctzk_hAYQI)

[ISSUE_KEY]: AWen1cRZC7Ctzk_hAYQI